stage1/init/common/mount.go

```diff
@@ -344,44 +347,38 @@ func AppAddOneMount(p *stage1commontypes.Pod, ra *schema.RuntimeApp, sourcePath
 	/* Prepare a temporary playground that is not a shared mount */
 	playgroundMount, err := ioutil.TempDir("", "rkt.propagate.")
 	if err != nil {
-		log.FatalE("error creating temporary propagation directory", err)
-		os.Exit(254)
+		return errwrap.Wrapf("error creating temporary propagation directory", err)
```

Comment:
Fun fact: these are the only uses of `errwrap.Wrapf` in this repository. I don't know if we were intentionally avoiding it because it does weird replace logic or not.